tests/integration/example_test_env_vars.yaml

```diff
@@ -11,3 +11,15 @@ post:
   ref: e1be527f39ec31323f0454f7d1422c6260b00580
   # Package managers
   pkg_managers: ["gomod"]
+# Test data for the Valid data in request test
+get:
```

Comment:
Why did you decided to make this test data configurable? It seems to me that these are static values and could be set in the tests themselves.

Reply:
The idea is that we can use different test data without having to update the code. It seems pretty powerful to me.